### PR TITLE
fix called class 'BlockPluginInterface' in token.module

### PR DESCRIPTION
**Pull request by rafarubert**

### CODE REVIEW

File: token.module [hunk 1]
```diff
@@ -182,7 +182,7 @@ function token_form_alter(&$form, $form_state, $form_id) {
 /**
  * Implements hook_block_view_alter().
  */
-function token_block_view_alter(&$data, \Drupal\block\BlockPluginInterface $block) {
+function token_block_view_alter(&$data, \Drupal\Core\Block\BlockPluginInterface $block) {
```
Comment (jibran) [inline token.module:185]: Please add use statement.

Comment (rafarubert) [inline token.module:185]: This update is necessary for install on drupal version "8.0.0-alpha14+187-dev (2014-Aug-26)" without error after installation. which drupal version are using?

Comment (jibran) [inline token.module:185]: I am not saying that this is an irrelevant change. I am just saying add `use Drupal\Core\Block\BlockPluginInterface;` on top of the file and only use `BlockPluginInterface $block` in function typehinting.

**Comment by Berdir** [comment]
https://github.com/larowlan/token/pull/29 includes this fix, and converts it to a use.

**Pull request closed without merging**